packages/perseus/src/widgets/lights-puzzle.tsx

```diff
@@ -105,17 +133,12 @@ class Tile extends React.Component<any> {
     }
 
     _flip = () => {
-        this.props.onChange(!this.props.value);
+        this.props.onChange({value: !this.props.value});
```

Comment:
This is a logic change, are we sure this is going to work as expected?

Reply:
Got approval to delete this widget!